.github/workflows/prepare-release.yaml

```diff
@@ -16,27 +12,13 @@ jobs:
   prepare-release:
     runs-on: ubuntu-latest
     steps:
-    #   - uses: actions/checkout@v4
```

Comment:
Would you mind uncommenting this instead of removing? @SergioGasquez discussed and we want to try and use it next release to fully automate the whole compiler build process.